conditionalIf.js

```diff
@@ -0,0 +1,20 @@
+
+
+function getGrade(score) {
+    let grade;
+    // Write your code here
+    if(score>25 && score<=30)
```

Comment:
too much if else block . refactor this method . this is not clean code.

Reply:
How to emove multiple if-else statements here

Reply:
use switch